scripts/hotrod-integration-test.sh

```diff
@@ -21,13 +47,15 @@ make create-baseimg
 
 # Loop through each platform (separated by commas)
 for platform in $(echo "$platforms" | tr ',' ' '); do
+  # Extract the operating system from the platform string
+  os=${platform%%/*}  #remove everything after the last slash
   # Extract the architecture from the platform string
   arch=${platform##*/}  # Remove everything before the last slash
-  make "build-all-in-one" GOOS=linux GOARCH="${arch}"
+  make "build-all-in-one" GOOS="${os}" GOARCH="${arch}"
```

Comment:
This is actually incorrect. This script is for building & testing hotrod. You are not actually reducing platforms on which the hotrod is build (L40-43). On the other hand, all-in-one does not need to be built for other platforms at all, only once for the current platform `"$(go env GOOS)/$(go env GOARCH)"`, because we only use it for e2e test and then discard.

Reply:
Okay so I should leave `build-all-in-one `with the single build just for linux and should bring back the implementation for hotrod, something like this >

`# Build for each platform
for platform in $(echo "$PLATFORMS" | tr ',' ' '); do
  os=${platform%/*}   # Extract OS (part before the slash)
  arch=${platform##*/} # Extract architecture (part after the slash)
  make build-examples GOOS="${os}" GOARCH="${arch}"
done`

Reply:
Yes the loop is for hotrod, allinone only needs to be built locally